tfe/resource_tfe_workspace_test.go

```diff
@@ -860,6 +864,8 @@ func TestAccTFEWorkspace_importVCSBranch(t *testing.T) {
 						"tfe_workspace.foobar", "vcs_repo.0.branch", GITHUB_WORKSPACE_BRANCH),
 					resource.TestCheckResourceAttr(
 						"tfe_workspace.foobar", "vcs_repo.0.ingress_submodules", "false"),
+					resource.TestCheckResourceAttr(
+						"tfe_workspace.foobar", "vcs_repo.0.tags_regex", ""),
```

Comment:
Likewise as above test comment. Could we add a test where we set tags_regex, and another test where we update it and delete it? Going through the full CRUD lifecycle.